Source/ARTMessage.m

```diff
@@ -45,3 +49,28 @@ - (NSInteger)messageSize {
 }
 
 @end
+
+@implementation ARTMessage (Decryption)
+
++ (instancetype)fromEncodedJsonObject:(NSDictionary *)input channelOptions:(ARTChannelOptions *)options error:(NSError **)error {
```

Comment:
it's just cosmetics, but the first parameter name in header declaration is `json` not `input`

Reply:
https://github.com/ably/ably-cocoa/pull/1307/commits/dd9a0ad288f1e64869e4977a735b69f6be95665e